From dd9599caaf73ab5561866046a335bfa5d5ecb7bf Mon Sep 17 00:00:00 2001
From: Alessandro Astone <ales.astone@gmail.com>
Date: Wed, 20 Mar 2019 21:19:29 +0100
Subject: [PATCH 08/10] common: recovery: allow setting time

Fixes:
 avc:  denied  { sys_time } for  pid=567 comm="recovery"
 capability=25 scontext=u:r:recovery:s0 tcontext=u:r:recovery:s0
 tclass=capability permissive=0

Change-Id: I3a5b2f5cc749fbee56c17765042b9dc616d4fedd

# Conflicts:
#	common/private/recovery.te
---
 common/private/recovery.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/private/recovery.te b/common/private/recovery.te
index f7dbcd5..291c960 100644
--- a/common/private/recovery.te
+++ b/common/private/recovery.te
@@ -47,4 +47,7 @@ allow recovery sysfs_batteryinfo:dir search;
 
 # Read fbe encryption info
 r_dir_file(recovery, unencrypted_data_file)
+
+# Set time
+allow recovery self:capability sys_time;
 ')
-- 
2.17.1

